instrumentation/github.com/gorilla/mux/otelmux/mux_test.go

```diff
@@ -162,3 +166,40 @@ func TestResponseWriterInterfaces(t *testing.T) {
 
 	router.ServeHTTP(w, r)
 }
+
+func TestCustomSpanNameFormatter(t *testing.T) {
+	exporter := tracetest.NewInMemoryExporter()
+	t.Cleanup(exporter.Reset)
```

Comment:
Is this cleanup necessary? The exporter will be in memory, and can't be reused by any other test. So once it has finished running, it will be garbage collected anyway.

Reply:
@dmathieu This is effectively a `exporter.Shutdown` (but without the annoying method signature making it impossible to `t.Cleanup`). I can remove it...